tests/test_mesh.py

```diff
@@ -294,6 +298,35 @@ def test_usd_parse(usd_filename):
         )
 
 
+@pytest.mark.skipif(sys.version_info[:2] != (3, 10), reason="USD Baking works only in Python == 3.10 now")
```

Comment:
It should be more restrictive. Skip if "not (Python == 3.10 and platform in (Linux, Windows)".